README.md

```diff
@@ -17,12 +17,15 @@ var Klarna = require('klarna-nodejs');
 
 var configuration =
 {
-	eid: 1,
-	sharedSecret: '...',
+    eid: 1,
+    sharedSecret: '...',
```

Comment:
It seems you use 4 spaces for indentation, but the existing code uses 1 tab. I'm all for using spaces instead of tabs, but then we should probably change all of the code, to keep consistency.